src/exoscale/compute/api/spec.clj

```diff
@@ -1,5 +1,17 @@
 (ns exoscale.compute.api.spec
-  (:require [clojure.spec.alpha :as spec]))
+  (:require [exoscale.compute.api.http :as http]
+            [clojure.spec.alpha :as spec]))
+
+(spec/def ::ne-string
```

Comment:
`(s/and string? (complement clojure.string/blank))` it avoid creating a seq and supports "        ".